### DISCUSSION TITLE
How can I change the tiltle color?

### DISCUSSION THREAD

**Discussion by SimoneFelici** [cotes2020/jekyll-theme-chirpy#1504] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I don't like the gray color of the title, so I tried to change the color with markdown, which worked, but then I noticed that in the tab it shows also the code and not just the title that I want, everyone knows how to fix this?
![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/57272391/64e95fee-b438-4138-b98b-ac06e3b00245)
![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/57272391/82f3acde-2c11-4e33-aec2-e5349b832837)

### Operations you have already tried

I tried searching for the style.css, but didn't found it.

**Comment by mohits** [comment, marked as answer]
The basic instructions are here: https://chirpy.cotes.page/posts/getting-started/#customizing-stylesheet

The theme is run from the gem, so you will find it in your gem installation under Ruby. Did you check there?

**Reply by SimoneFelici** [reply]
Oh sorry, I actually figure it out but forgot to remove the post, thanks

**Reply by SimoneFelici** [reply]
Actually now I have another problem, I edited all the thing that I wanted, Aparte from the Trending Tags, when I hover on a trending tag is blue, how do I change that? I tried searching for it but I got nothing

**Reply by mohits** [reply]
Still sounds like something to update on the SCSS but I'm not close to it, so I can't help you much, sorry!

**Reply by SimoneFelici** [reply]
Yeah, I think that too, but I really can't find it lol, I think that I am going to remade an issue with this new thing that I have encountered, thanks for the help.

**Reply by NichtsHsu** [reply]
You may check `--bs-btn-hover-bg` and `--bs-btn-hover-border-color`.
